phoenix-postgres-pgadmin/example_app/Dockerfile

```diff
@@ -0,0 +1,65 @@
+
+# ---- Build Stage ----
+FROM elixir:1.9-alpine AS app_builder
+
+# Set environment variables for building the application
+ENV MIX_ENV=prod
+ENV LANG=C.UTF-8
+
+# install build dependencies
+RUN apk add --update git build-base nodejs npm yarn python
+
+# Install hex and rebar
+RUN mix local.hex --force && \
+    mix local.rebar --force
+
+# Create the application build directory
+RUN mkdir /app
+WORKDIR /app
+
+# install mix dependencies
+COPY mix.exs mix.lock ./
+COPY config config
+RUN mix deps.get
+RUN mix release.init
+RUN mix deps.compile --force --only $MIX_ENV 
```

Comment:
Maybe we can group this 3 commands to only have one layer? 
I suppose they are all depending of the previous `COPY` command?
```dockerfile
RUN mix deps.get \
    && mix release.init  \
    && mix deps.compile --force --only $MIX_ENV
```

Reply:
Changed as you rightly pointed out